src/browser/settings.jsx

```diff
@@ -33,6 +38,12 @@ var SettingsPage = React.createClass({
       showAddTeamForm: false
     });
 
+    appLauncher.isEnabled().then(function(enabled) {
```

Comment:
@yuya-oc doesn't seem to work :(
![image](https://cloud.githubusercontent.com/assets/5943908/16017762/a4ec70ca-31a1-11e6-9744-0d1034a10757.png)
I'll push that iteration of the code, anyway